client/landing/stepper/constants.ts

```diff
@@ -1,11 +1,11 @@
 import type { Design } from '@automattic/design-picker';
 
 export const WRITE_INTENT_DEFAULT_DESIGN: Design = {
-	slug: 'livro',
-	title: 'Livro',
+	slug: 'hey',
+	title: 'hey',
```

Comment:
[nit] 
```suggestion
	title: 'Hey',
```

Reply:
I noticed this too but this does not affect anything as it's not used anywhere 🙈